### ISSUE TITLE
Visualizing Level Sets

### ISSUE THREAD

**Issue by avaxman** [libigl/libigl#1393]
#### Describe your issue

The (Really nice) level set visualization given in the tutorial example 206 (Exact Discrete Geodesic Distances) is not what you get when running the tutorial. One gets something like this:

<img width="1359" alt="Screenshot 2019-12-25 at 11 03 14" src="https://user-images.githubusercontent.com/8985720/71442433-31fa5d80-2706-11ea-8330-d2eee3ae342e.png">

This is done by color interpolation, and therefore not very applicable to coarse meshes and has interpolation artifacts. Does libigl provide something like the original drawing? maybe something based on texture or shader?

#### Check all that apply (change to `[x]`)
- [ ] Windows
- [X] Mac OS X
- [ ] Linux
- [ ] I have tried the `dev` branch and the problem persists

See https://libigl.github.io/CONTRIBUTING/#bugreport for more tips.

**Comment by jdumas** [comment]
In the `dev` branch we switched to interpolating colors from a 1D texture in the fragment shader. This is what's shown on the website. Hopefully the discrepancy will be fixed for the next release of libigl :)

**Comment by avaxman** [comment]
Meanwhile I discovered a “libigl::isolines” which is used in Example 712, albeit undocumented. It apparently follows the Directional/libhedra paradigm of creating actual geometry for plotting lines.

Amir.

**Comment by jdumas** [comment]
Yes. I remember the `igl::isolines` has some issues (see #1245) that haven't been fixed yet. In the `dev` branch we switched to `igl::isolines_map` to inject the isolines in the colormap texture, which is why `igl::isolines` doesn't appear in the tutorials on the website.

Injecting the isolines into the colormap will indeed be problematic for coarse meshes and whatnot, so maybe we should default to explicit overlay lines. That being said one can also do tricks in the fragment shader to control the screen-space size of an isoline using a separate texture containing just the isoline information.

Bottom line is: you have two ways of plotting isolines now in the `dev` branch, `igl::isolines` gives you explicit geometry, while `igl::isolines_map` will inject isolines into a 1d colormap texture whici is used to interpolate vertices colors in the fragment shader.

**Comment by avaxman** [comment]
I see. In the meantime I indeed just used overlay meshes:

<img width="982" alt="Screenshot 2019-12-25 at 12 21 28" src="https://user-images.githubusercontent.com/8985720/71444378-19dc0b80-2711-11ea-8f0b-c6c7e69fd02f.png">

**Comment by jdumas** [comment]
Sounds good. I'll close the issue then, feel free to re-open if you have another question.
